sys/src/9/386/pci.c

```diff
@@ -690,11 +733,17 @@ pciclrmwi(Pcidev* p)
 	pcicfgw16(p, PciPCR, p->pcr);
 }
 
+/*
+ * DMG 06/11/2016. Move the capability offset finding in a separate function
```

Comment:
It's recommended in these projects that you not put names and dates. I realize we have some names in various places but that's historical stuff we don't want to remove. But if it's OK can you remove the DMG and date here. If this is a TODO, mark it as a TODO. If it is describing current code, then it would help to just write a description of the function.